src/main/java/Executer.java

```diff
@@ -0,0 +1,35 @@
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+public class Executer {
+    private String databasePath;
+    private String pythonCaller;
+    private static Executer executerObject = null;
+    private Executer() throws IOException {
+        Path filePath = Paths.get(Paths.get(System.getProperty("user.dir")).toString(), "src", "main", "java", "configurations.properties");
+        loadConfg(filePath.toString());
+    }
+    public static Executer getInstance() throws IOException {
+        if(executerObject == null){
+            executerObject = new Executer();
+        }
+        return executerObject;
+    }
+    private void loadConfg(String confgFilePath) throws IOException {
+        FileReader confgFile = new FileReader(confgFilePath);
+        Properties confg = new Properties();
+        confg.load(confgFile);
+        databasePath = confg.getProperty("databasePath");
```

Comment:
Move those to constructor and make them final

Reply:
i made those in a function and call it in the constructor to make it clear.
should I delete this function and put the code in the constructor

Reply:
No, make this function return config 
In constructor, make database path final 
This.databasepath = config.get